src/Microsoft.OpenApi.OData.Reader/Generator/OpenApiResponseGenerator.cs

```diff
@@ -165,6 +180,61 @@ public static OpenApiResponses CreateResponses(this ODataContext context, IEdmOp
             return responses;
         }
 
+        private static OpenApiResponse CreateCollectionResponse(IEdmStructuredType structuredType)
+        {
+            var entityType = structuredType as IEdmEntityType;
+            return CreateCollectionResponse(entityType?.FullName() ?? structuredType.FullTypeName());
+        }
+        private static OpenApiResponse CreateCollectionResponse(string typeName)
```

Comment:
It could be remove this private static method and put the codes directly into the above?

Reply:
No because this method is called twice